contents/blog/startup-unified.md

```diff
@@ -0,0 +1,55 @@
+
+### What sort of insights did you get from your customers?
+“One was regarding our pricing model. The value metric that you use for pricing is really quite important. It's not the actual price, because there are obvious connotations there if you’re priced too high or too low, but the actual metric. We used to charge per connection, per account, but we discovered that users wanted more flexibility. So, we now charge based on how often users hit our API. 
+
+That was super important because it allowed our customers a lot of flexibility in terms of how often they synchronize customers' data, and is basically a one-to-one correspondence with how much they charge _their_ customers. If they're charging their customer a lot, then real-time synchronization is fine. If you're on a free plan or a very inexpensive plan, you probably don't want to do that. You'll probably want to do a daily synchronization, so that was really key.
+
+The second thing we learned from our user interviews is that there was a specific need from HR software companies. We initially thought we’d focus on sales-oriented software companies, but nearly all the customers coming in were HR software teams. So, we tweaked the roadmap to reflect their interest.”
+
+### Looking forward, what are you most excited about?
+“I'm a big data nerd and I geek out most on seeing what users are doing on my platform. I insist our teams don’t build anything they can’t measure, and that’s where PostHog comes in. I’m really excited to see the funnels, customer journeys, and understand how they change as we built. Then you can do a bunch of analysis and understand how your product _is_ being used instead of _assuming_ how it’s used.
+
+This sort of data makes your product better. It allows us to tweak our messaging as well, tweak the product, and generally make a lot of improvements. 
+
+I'm also an API nerd — and this industry is all about APIs. The engineers that I work with, the marketers, everybody loves APIs because we're dealing with them all the time. I don't feel like APIs are going away. They're not shrinking, they're just expanding. Everybody has APIs. I think that's quite exciting.
+
+### On the other hand, what are you most worried about?
+“Well, I’m _not_ worried about AI! My last startup started off as an AI chatbot eight years ago, in the last hype cycle of AI chat. I saw people asking me when the AI robots were going to kill us and all that, and that was eight years ago. We're getting the exact same thing now. AI now is a thousand times more powerful, but it’s still not general AI.
+
+What I _do_ worry about is funding. We're going through a really weird and unique cycle in tech funding right now, of the type not seen since the dot com crash in the 2000s. Smart investors are putting money elsewhere, thanks to Covid, so that's causing a ripple effect throughout the entire market.
```

Comment:
...thanks to high interest rates...

Reply:
```suggestion
What I _do_ worry about is funding. We're going through a really weird and unique cycle in tech funding right now, of the type not seen since the dot com crash in the 2000s. Smart investors are putting money elsewhere, thanks to high interest rates, so that's causing a ripple effect throughout the entire market.
```